Sources/MapboxMaps/Foundation/MapViewDependencyProvider.swift

```diff
@@ -93,7 +94,9 @@ internal final class MapViewDependencyProvider: MapViewDependencyProviderProtoco
     func makeAnimationLockoutGestureHandler(view: UIView,
                                             mapboxMap: MapboxMapProtocol,
                                             cameraAnimationsManager: CameraAnimationsManagerProtocol) -> GestureHandler {
-        let gestureRecognizer = AnyTouchGestureRecognizer()
+        let gestureRecognizer = ContinuousAnyTouchGestureRecognizer(
+            minimumPressDuration: 0.15,
```

Comment:
How is the 0.15 defined? is it the threshold that we recognise the double tap gesture?

Reply:
It's how long touches must remain on the screen before animations will begin to be cancelled.